File: negacio.py
# Negáció igazságtáblázata

def neg(x):
    a = bool(x)
    c = not a
    return int(c)

print("A  ⌝A")
print("-----")

for i in range(2):   # 0 és 1
    k = neg(i)
    print(f"{i}  {k}")

print("-----")

File: negacio.js
// Negáció igazságtáblázata

function neg(x) {
  var a = Boolean(x);
  var c = !a;
  return Number(c);
}

console.log("A  ⌝A");
console.log("-----");

for (var i = 0; i <= 1; i++) {
  var k = neg(i);
  console.log(i + "  " + k);
}

console.log("-----");
